lib/hashes.md

```diff
@@ -5,15 +5,16 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a Hash?
 
-Replace me with your response...
+Hashes are sort of like dictionaries. It is a collection of keys and values where you can refer to a value using the associated key, or vice versa.
```

Comment:
Nicely done! That's a good explanation.

You are correct that finding all keys for a given value is not a common way of using hashes, and that's why there isn't a readymade way to do so.